tests/utils.py

```diff
@@ -104,12 +104,31 @@ async def assert_revert_entry_point(fun, invalid_selector):
     await assert_revert(fun, entry_point_msg)
 
 
-def assert_event_emitted(tx_exec_info, from_address, name, data):
-    assert Event(
-        from_address=from_address,
-        keys=[get_selector_from_name(name)],
-        data=data,
-    ) in tx_exec_info.raw_events
+def assert_event_emitted(tx_exec_info, name, data, order=0):
+    """Assert one single event is fired with correct data."""
+    assert_events_emitted(tx_exec_info, [(order, name, data)])
```

Comment:
Once we can use `invoke` in the testing framework, I'm assuming we can use the old logic with the returned `Event` object (with the `from_address`) instead of the `OrderedEvent`. This is just for the transition until then.

We can add an assertion line in this function checking the from-address

Reply:
if it's not too much work, it'd be nice to have full coverage even if it's temporal